OSX/patches/allegro-4.4.2.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 245be10..105c923 100644
+index 245be10fb..6147a68b3 100644
```

Comment:
is this patch still required now that allegro 4.4.3 is released ?

Reply:
For macos it is, the patch includes mac specific changes (and 4.4.3)